snapcraft.yaml

```diff
@@ -1628,10 +1641,19 @@ parts:
       ln -s gettext-0.19.8 usr/share/gettext-current
       ln -s gdk-pixbuf-2.0/2.10.0 usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/gdk-pixbuf-current
 
-      # Fix dangling symlink by overwriting it
-      ln -sf lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/libc_malloc_debug.so.0 usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/libc_malloc_debug.so
-      # CHECK THIS When changing the core base, this line must be update
-      cp /snap/$CRAFT_EXT_CORE_LEVEL/current/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/libc_malloc_debug.so.0 lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/
+      # Check dangling libraries symlinks
+      invalid_links=()
+      core_root_prefix=/snap/$CRAFT_EXT_CORE_LEVEL/current
+      for i in $(find "$CRAFT_PRIME"/{usr/,}lib/{"$CRAFT_ARCH_TRIPLET_BUILD_FOR",} -xtype l -name '*.so'); do
```

Comment:
Given that the Find command is recursive, this is equivalent to `$CRAFT_PRIME/{usr/,}lib`.

Reply:
Oh, right... Indeed!